python/drisk_api/graph_client.py

```diff
@@ -291,24 +289,45 @@ def create_view(
             UUID: The ID of the created view node.
 
         """
-        view_node = self.create_node(label=label)
-        if x_node is None:
-            x_node = self.create_node(label="x ")
-        if y_node is None:
-            y_node = self.create_node(label="y ")
+        with self.batch():
+            view_node = self.create_node(label=label)
+            if x_node is None:
+                x_node = self.create_node(label="x ")
+            if y_node is None:
+                y_node = self.create_node(label="y ")
 
-        self.create_edge(self.graph_id, view_node, 1.0)
-        self.create_edge(view_node, x_node, 0.0)
-        self.create_edge(view_node, y_node, 1.0)
+            self.create_edge(self.graph_id, view_node, 1.0)
+            self.create_edge(view_node, x_node, 0.0)
+            self.create_edge(view_node, y_node, 1.0)
 
-        if filters is not None:
-            for i, f in enumerate(filters):
-                self.create_edge(view_node, f, i + 2.0)
+            if filters is not None:
+                for i, f in enumerate(filters):
+                    self.create_edge(view_node, f, i + 2.0)
 
         return view_node
 
+    def get_view_axes_and_filters(
+        self, view_node: str
+    ) -> Tuple[str, str, List[str]]:
+        """
+        Get the x-axis, y-axis, and filter nodes of a view.
+
+        Args:
+        ----
+            view_node (str): The ID of the view node.
+
+        """
+        x_node, y_node, *filters = [
+            uuid
+            for uuid, _ in sorted(
+                zip(*self.get_successors(view_node, weights=True)),
```

Comment:
if easy extracting this out as `get_sorted_successors` could certainly be useful (I know I've done this before)